ci/test.yml

```diff
@@ -194,3 +194,32 @@ rollout test integration manual:
   script:
     - yarn install --frozen-lockfile --cache-folder .yarn --prefer-offline
     - yarn workspace @trezor/rollout test:integration
+
+# @trezor/transport
+
+# todo: this works locally. does it work in CI?
+transport test with simple script:
+  stage: integration testing
+  before_script:
+    - yarn install --frozen-lockfile --cache-folder .yarn --prefer-offline
+    - yarn workspace @trezor/transport build:lib
+  script:
+    - ./docker/docker-transport-test.sh
+
+# todo: compose might be overkill here
+transport with compose:
+  stage: integration testing
+  variables:
+    COMPOSE_FILE: ./docker/docker-compose.transport-ci.yml
+  script:
+    - yarn install --pure-lockfile --cache-folder .yarn --prefer-offline
+    - yarn workspace @trezor/transport build:lib
+    - docker-compose pull
+    - docker-compose up -d trezor-user-env-unix
+    - sleep 5
+    - git submodule add --force https://github.com/trezor/trezor-common.git trezor-common
+    - yarn pbjs -t json -p ./trezor-common/protob -o ./packages/integration-tests/projects/transport/messages.json --keep-case messages-bitcoin.proto messages-bootloader.proto messages-common.proto messages-crypto.proto messages-debug.proto messages-management.proto
```

Comment:
can we launch this from the script `docker/docker-transport-test.sh` so we don't have this long command in yml file?

Reply:
thats what I would like to do. There are intentionally two approaches that do the same - one starting on 201 and the other one starting on 210

I would like to use 201 line approach but uses `./docker/run-next-to-tuenv.sh` script which basically runs the test script itself in the base image but with the support of trezor-user-env whose ports should be exposed into the base image. But somehow this does not work for me and trezor-user-env was not reachable from tests.